.lintr

```diff
@@ -0,0 +1,12 @@
+linters: with_defaults(
+  line_length_linter(120), 
+  commented_code_linter = NULL,
+  object_name_linter = object_name_linter("camelCase"),
+  open_curly_linter = NULL,
+  closed_curly_linter = NULL,
+  object_usage_linter = NULL
```

Comment:
Also looks like it can be left enabled.

Reply:
If it's left enabled, I get the following errors:
```r
R/util.R:91:11: warning: no visible binding for global variable ‘lastError’
  pkg.env$lastError
          ^~~~~~~~~
R/util.R:146:16: warning: no visible binding for global variable ‘_id’
  val <- vals$`_id`
               ^~~
```

The first I can fix by adding `pkg.env$lastError <- NULL` at the top of `util.R`. But one of the biggest reasons to use `lintr` is for IDE integration, imho, and unfortunately any package-internal functions (i.e. those that aren't included in the `roxygen`-generated `NAMESPACE`) will threw an error (see https://github.com/jimhester/lintr/issues/482).

Additionally, `object_usage_linter` executes package code with `eval` during checks, which is potentially a bit dangerous (although there aren't any other contributors to the package, so it's probably not a huge concern).

Reply:
Okay, fine to leave it then.

(I'd like to get rid of `getLastError` btw. R has better facilities with try/catch I think.)

Just for my understanding, is the second one a false-positive? We have other code like getStatistics.R:161 ```percentOf <- serverPops$`_id` ```